Conventus/InConceptioneBMV/resp-hodieconceptaest-CROCHU.gabc

```diff
@@ -7,14 +7,14 @@ transcriber: Jakub Jelínek;
 nabc-lines: 1;
 %%
 (c4)
-HO(ghOi|sa)di(j|vi)e(iiOh|pr-) *()
-con(i|vi)cép(i)ta(ihhOg/h|cl!pivihg) est(hg|cl-) (`)
+HO(ghoi|sa)di(j|vi)e(iiOh|pr-) *()
```

Comment:
Salicus should be ghOi rather than ghoi, but on the other side the pressus should be iioh rather than iiOh, no?